server/api/playbook_runs.go

```diff
@@ -89,6 +89,8 @@ func NewPlaybookRunHandler(
 	playbookRunRouterAuthorized.HandleFunc("/update-description", withContext(handler.updateDescription)).Methods(http.MethodPut)
 	playbookRunRouterAuthorized.HandleFunc("/restore", withContext(handler.restore)).Methods(http.MethodPut)
 	playbookRunRouterAuthorized.HandleFunc("/actions", withContext(handler.updateRunActions)).Methods(http.MethodPut)
+	playbookRunRouterAuthorized.HandleFunc("/status-update/enable", handler.enableStatusUpdate).Methods(http.MethodPut)
+	playbookRunRouterAuthorized.HandleFunc("/status-update/disable", handler.disableStatusUpdate).Methods(http.MethodPut)
```

Comment:
you will need the withContext wrapper to add context as the first parameter of the endpoint (latest changes in log management).
The you'll have to get from there the logger to inject it into the handleError func (second parameter)

```suggestion
	playbookRunRouterAuthorized.HandleFunc("/status-update/disable", withContext(handler.disableStatusUpdate)).Methods(http.MethodPut)
```